src/main/java/com/spotify/sparkey/ByteBufferCleaner.java

```diff
@@ -47,11 +47,7 @@ public void clean(final MappedByteBuffer byteBuffer) {
         if (cleaner != null) {
           cleaner.getClass().getMethod("clean").invoke(cleaner);
         }
-      } catch (IllegalAccessException e) {
-        throw new RuntimeException(e);
-      } catch (InvocationTargetException e) {
-        throw new RuntimeException(e);
-      } catch (NoSuchMethodException e) {
+      } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
         throw new RuntimeException(e);
```

Comment:
Consider adding a message for context here (I know it's unrelated to this change)